project/api/views.py

```diff
@@ -122,6 +124,69 @@ class SpacetimeVolumeViewSet(viewsets.ModelViewSet):
     )
     serializer_class = SpacetimeVolumeSerializer
 
+    @transaction.atomic
+    def create(self, request, *args, **kwargs):
+        """
+        Solve overlaps if included in request body
+        """
+
+        geom = GEOSGeometry(str(request.data["territory"]))
+        start_date = float(request.data["start_date"])
+        end_date = float(request.data["end_date"])
+        overlaps_db = SpacetimeVolume.objects.raw(
```

Comment:
This query will be executed each time, correct?
Can we cache this query somehow? Maybe we can use redis or something else